spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/AppDeploymentRequestCreator.java

```diff
@@ -225,20 +231,22 @@ private String extractAppVersionProperty(StreamAppDefinition appDefinition, Map<
 	/* default */ Map<String, String> extractAppProperties(StreamAppDefinition appDefinition,
 			Map<String, String> streamDeploymentProperties) {
 		Map<String, String> appDeploymentProperties = new HashMap<>();
-		// add common properties first
-		appDeploymentProperties.putAll(this.commonApplicationProperties.getStream());
-		// add properties with wild card prefix
-		String wildCardProducerPropertyPrefix = "app.*.producer.";
-		String wildCardConsumerPropertyPrefix = "app.*.consumer.";
-		String wildCardPrefix = "app.*.";
-		parseAndPopulateProperties(streamDeploymentProperties, appDeploymentProperties, wildCardProducerPropertyPrefix,
-				wildCardConsumerPropertyPrefix, wildCardPrefix);
-		// add application specific properties
-		String producerPropertyPrefix = String.format("app.%s.producer.", appDefinition.getName());
-		String consumerPropertyPrefix = String.format("app.%s.consumer.", appDefinition.getName());
-		String appPrefix = String.format("app.%s.", appDefinition.getName());
-		parseAndPopulateProperties(streamDeploymentProperties, appDeploymentProperties, producerPropertyPrefix,
-				consumerPropertyPrefix, appPrefix);
+
+		if (appDefinition.getApplicationType() != ApplicationType.app) {
+			appDeploymentProperties.putAll(this.commonApplicationProperties.getStream());
```

Comment:
I believe we may still want to support app deployment properties as this method is being used in other places where we want to extract the deployment properties (other than partitioning and binding specific properties)